### ISSUE TITLE
Crash on Change OU

### ISSUE THREAD

**Issue by iamtheyammer** [iamtheyammer/gam-cros-win-wrapper#4]
error from event viewer:
```
Application: ChromebookGUI.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ArgumentOutOfRangeException
```

**Comment by iamtheyammer** [comment]
This issue was attempted to be fixed in commit:bc2e2f7 but we'll see if it is completely fixed.

**Comment by iamtheyammer** [comment]
Not fixed in latest commit, will continue debugging.
